packages/react/src/use-completion.ui.test.tsx

```diff
@@ -79,34 +79,37 @@ describe('stream data stream', () => {
   });
 
   describe('loading state', () => {
+    const server =createTestServer({
+      '/api/completion': {}
+   });
+
```

Comment:
could pull out server const to top level and use `server.url[..]=` in individual tests

Reply:
good idea!